Description: Don't include remote scripts
 We have them in Debian, use those
 sed -i \
   -e 's,{giacjsonline},{giacjshere},g' \
   -e 's,http://cdn.mathjax.org/mathjax/latest/,file:///usr/share/javascript/mathjax/,g' \
   -e '/^%\loadgiacjsonline/d' \
   doc/testfr.tex doc/test.tex doc/giac.tex doc/giacfr.tex doc/xcasen.html doc/xcasfr.html
 then refresh, pop, and manually remove false positives from this patch, e.g.
 %%comments and \verb literals and \newenvironment declarations.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/test.tex
+++ b/doc/test.tex
@@ -59,9 +59,8 @@
 \makeindex
 
 \begin{document}
-%\loadgiacjsonline
 %\loadgiacjs
-\begin{giacjsonline}
+\begin{giacjshere}
 \maketitle
 
 \tableofcontents
@@ -164,5 +163,5 @@
 \verb|\giacslider{a}{-5}{5}{0.1}{0}{plot(sin(a*x))}|\index{giacslider}\\
 \giacslider{a}{-5}{5}{0.1}{0}{plot(sin(a*x))}
 
-\end{giacjsonline}
+\end{giacjshere}
 \end{document}
--- a/doc/giac.tex
+++ b/doc/giac.tex
@@ -348,7 +348,7 @@
   if (UI.usemathjax){
     var script = document.createElement("script");
     script.type = "text/javascript";
-    script.src  = "http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
+    script.src  = "file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
     document.getElementsByTagName("head")[0].appendChild(script);
   }
   var elem= document.getElementById('controlindex');
--- a/doc/giacfr.tex
+++ b/doc/giacfr.tex
@@ -342,7 +342,7 @@
   if (UI.usemathjax){
     var script = document.createElement("script");
     script.type = "text/javascript";
-    script.src  = "http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
+    script.src  = "file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
     document.getElementsByTagName("head")[0].appendChild(script);
   }
   var elem= document.getElementById('controlindex');
--- a/doc/testfr.tex
+++ b/doc/testfr.tex
@@ -77,7 +77,6 @@
 \begin{document}
 \begin{giacjshere}
 %\loadgiacjs
-%\loadgiacjsonline
 Un $x$
 $$ 1/2 $$
  
